native_client/javascript/README.md

```diff
@@ -1,4 +1,4 @@
-Full project description and documentation on GitHub: [https://github.com/mozilla/DeepSpeech](https://github.com/mozilla/DeepSpeech).
+Full project description and documentation on [https://deepspeech.readthedocs.io/](https://deepspeech.readthedocs.io/).
```

Comment:
Not a note on this file but hasn't native_client/java/README.md been forgotten?

Reply:
The Java README is more akin to the native_client README or the dotnet README, where it's mostly about development and building rather than using of pre-built binaries. We need to write the usage instructions for them, and then add those to RTD too.